logstash-devutils.gemspec

```diff
@@ -35,12 +37,12 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "rake" # MIT License
   spec.add_runtime_dependency "gem_publisher" # MIT License
   spec.add_runtime_dependency "minitar" # GPL2|Ruby License
-  spec.add_runtime_dependency "logstash-core-plugin-api", "<= 2.99", ">= 2.0"
 
-  # Should be removed as soon as the plugins are using insist by their
-  # own, and not relying on being required by the spec helper.
-  # (some plugins does it, some use insist throw spec_helper)
-  spec.add_runtime_dependency "insist", "1.0.0" # (Apache 2.0 license)
+  spec.add_runtime_dependency "logstash-core", ">= 6", "< 9.0"
```

Comment:
not a good idea - should remove since the core-plugin-api has a fixed logstash-core 
(or ditch plugin-api dep)